.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: minimal
+
+services: docker
+
+env:
+  global:
+    - COMPOSE_PROJECT_NAME=articlestore
+
+install:
+  - ENV=dev make build
```

Comment:
environment is problematic as this term, even if `NODE_ENV` is probably a standard here and there. These are targets or variants but there are many more environments in which an identical `prod` image could run.

Reply:
the only standard I can find here is from `docker build`: `TARGET=dev`

Reply:
Hmm, not sure 'target' is the best, but let's go with it for now.